examples/criteo_benchmark.py

```diff
@@ -146,7 +146,7 @@ def parse_args():
 # TODO: Implement the get_embedding_size for dask-based workflow
 embeddings = [
     x[1]
-    for x in get_embedding_size(proc.stats["categories"], proc.columns_ctx["categorical"]["base"])
+    for x in get_embeddings(proc)
```

Comment:
I think this is called in a couple of other spots (gpu_benchmark.ipynb, criteo-example.ipynb, ossmann-store-sales-example.ipynb). Can you update those too?